pkg/proxy/nftables/proxier.go

```diff
@@ -336,6 +340,7 @@ var nftablesBaseChains = []nftablesBaseChain{
 	{"filter-input", knftables.FilterType, knftables.InputHook, knftables.DNATPriority + "-10"},
 	{"filter-forward", knftables.FilterType, knftables.ForwardHook, knftables.DNATPriority + "-10"},
 	{"filter-output", knftables.FilterType, knftables.OutputHook, knftables.DNATPriority + "-10"},
+	{"filter-output-post-dnat", knftables.FilterType, knftables.OutputHook, knftables.FilterPriority},
```

Comment:
We may want to reconsider the chain names. At any rate, at this point you need to remove the comment on line 328 that claims that `All of our base chains have names that are just "${type}-${hook}".`

Reply:
for consistency should we follow ${type}-${hook}-${priority} ?

Reply:
that makes sense, except for the fact that most of the priorities are negative numbers so you'd end up with double hyphens... but something along those lines might be good. You could just use "early" "normal" "late" or something like that?

Reply:
how about "${type}-${hook}-(before|at|after)-${target}" ? eg. `filter-postrouting-after-dnat` and `filter-prerouting-before-dnat`

"early" "normal" "late" will revolve around `dnat`, we also use `snat` priority for nat-postrouting.
Maybe we can define priority around a target - snat | dnat ?